govuk_publishing_components.gemspec

```
@@ -29,12 +29,12 @@ Gem::Specification.new do |s|
   s.add_development_dependency "capybara", "~> 3.25"
   s.add_development_dependency "foreman", "= 0.85"
   s.add_development_dependency "gds-api-adapters", "~> 59.1"
-  s.add_development_dependency "govuk-lint", "~> 3.11"
   s.add_development_dependency "govuk_schemas", "~> 3.2"
   s.add_development_dependency "govuk_test", "~> 1"
   s.add_development_dependency "jasmine", "~> 3.4.0"
   s.add_development_dependency "pry-byebug"
   s.add_development_dependency "rspec-rails", "~> 3.8"
+  s.add_development_dependency "rubocop-govuk", "~> 2"
```

```suggestion
  s.add_development_dependency "rubocop-govuk", "~> 3"
```

3.0.0 is the latest, released yesterday, and pulls in yet more RuboCop rules (and some improvements!)